WIP-CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Made dex ID Token expiration time configurable
 - Made dex tokens expiry times configurable
 - Excluded the `gatekeeper-system` namespace from velero backups in the workload cluster.
+- Rclone can now be configured to run every x minutes/hours/days/week/month/year.
```

Comment:
I would consider this as a fix. :slightly_smiling_face:

Reply:
fixed